WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,228 @@
+
+Examples of topics that should use the WEP process:
+
+- Adjusting the structure of the Slack community to make it more accessible to new community members
+- Policies about the structure of the Write the Docs organization
+- Policy related to Slack team or channel guidelines
+- Changes to the Code of Conduct (CoC)
+- A grant policy to support conference attendees that need financial support
+- A policy on how to balance various topics submitted to the Call for Proposals at the various Write the Docs conferences
+
+Examples of topics that are out of the scope of the WEP process:
+- Requests to change CFP policies in a specific conference
+- Requests to create a new Slack channel
+- Specific CoC violation reports
+- Requests to create a new meetup
+
+Topics that are out of scope for a WEP should be discussed in other community platforms, such as within the conference team, Slack team, or Code of Conduct team. The WEP team will direct out-of-scope pre-proposals to the relevant team in the rejection comments.
+
+Generally, WEPs do not contain the names of individual people, but rather references to teams in the Write the Docs organization. Similarly, they don’t contain fixed budget numbers, but instead guidelines for how to implement the budget. 
+
+How to submit a WEP
+-------------------
+
+Anyone from the community can submit a WEP! Here's how it works, and what to expect.
+
+The WEP process and system are administered by the following teams:
+
+Core Operations team
+  There are several reference in this WEP to the **core operations (CoreOps) team**. This team is currently known as the **core team** and the team description can be found on the `Core team page <https://www.writethedocs.org/team/#core-team>`_. The updated name is intended to reflect the purpose of the team more accurately in the context of its duties for the community. 
+
+WEP team
+  The **WEP team** is a new team that reviews WEP proposals for clarity and completeness. For the purpose of this WEP the team is initially composed of the Core Operations team, but in the the future will be open for other community members to join. 
+  
+Note that the process to join and leave teams is out of scope of this WEP and will be handled in a future WEP.
+
+At a very high level, the WEP submission process includes the following stages:
+
+#. `Pre-proposal`_ — Someone has an idea and they start collecting early input and feedback to see if it is within the scope of a WEP. This can be done informally, publicly or privately.
+
+#. `Draft submission and pre-review`_ — The WEP author writes a rough draft of the WEP and submits it in a pull request (PR) to the WEPs repository. The draft is pre-reviewed by an editor for style, formatting, and eligibility for the WEP process. If the WEP is pre-approved, it is accepted for discussion and the review period begins.
```

Comment:
Is it worth stating '30-day review period' here? My immediate reaction was to question how long this was... I thought I'd missed something until I found out 40 lines later.

Reply:
Probably. The 30 days only starts after the draft period, and it is important that that is clear.

Reply:
I've clarified this.